SPECS/cri-tools/CVE-2023-45288.patch

```diff
@@ -0,0 +1,1554 @@
+From d18ca301345ebac40ff5d53f7e154796b2e51fb9 Mon Sep 17 00:00:00 2001
+From: "dependabot[bot]" <49699333+dependabot[bot]@users.noreply.github.com>
+Date: Wed, 3 Jul 2024 14:24:11 +0000
+Subject: [PATCH] Bump google.golang.org/grpc from 1.64.0 to 1.65.0
```

Comment:
It seems to me that the CVE-2023-45288 is a net/http2 frame related CVE.
Double checking if the patch applied is the actual solution of the CVE.

In my opinion, a patch of the `vendor/golang.org/x/net/http2/` should be enough :)

Also, as per ASTROLABE, the affected files do not show `google.golang.org/grpc`.